web/_pages/index.html

```diff
@@ -8,7 +8,7 @@ <h1 id="home">Making X.509 errors usable.</h1>
     <div class="lead">
     <p>Validating X.509 certificates correctly turns out to be pretty complicated (e.g. <a href="http://www.cs.utexas.edu/~shmat/shmat_ccs12.pdf">Georgiev2012</a>, <a href="https://crocs.fi.muni.cz/papers/acsac2019">Ukrop2019</a>). Yet certificate validation is absolutely crucial for secure communication on the Internet (think <a href="https://en.wikipedia.org/wiki/Transport_Layer_Security">TLS</a>).</p>
     <p>Our goal is to simplify the ecosystem by consolidating the errors and their documentation (similarly to <a href="https://blog.mozilla.org/blog/2017/10/18/mozilla-brings-microsoft-google-w3c-samsung-together-create-cross-browser-documentation-mdn/">web documentation</a>) and by explaining better what the validation errors mean.</p>
-    <p>For every error, we aim to provide an example certificate ({% include icon.html icon="certificate" %}), documentation from OpenSSL ({% include icon.html icon="openssl-book" %}) and other TLS libraries ({% include icon.html icon="gnutls-book" %}, {% include icon.html icon="botan-book" %}, {% include icon.html icon="mbed-book" %}). In the future, we plan the possibility of reorganization based on the other libraries (currently, the web is organized by OpenSSL), adding the error frequencies based on IP-wide scans and elaborating on the consequences of individual errors.</p>
+    <p>For every error, we aim to provide an example certificate ({% include icon.html icon="certificate" %}), documentation from OpenSSL ({% include icon.html icon="openssl-book" %}, followed by {% include icon.html icon="unused-openssl-icon" %} if that error is not used anymore) and other TLS libraries ({% include icon.html icon="gnutls-book" %}, {% include icon.html icon="botan-book" %}, {% include icon.html icon="mbed-book" %}). In the future, we plan the possibility of reorganization based on the other libraries (currently, the web is organized by OpenSSL), adding the error frequencies based on IP-wide scans and elaborating on the consequences of individual errors.</p>
```

Comment:
Let's be a bit more concise and precise in the parenthesis: "<book>, deprecated or unused denoted by <cross>".

Reply:
Second note: Be precise in the naming. Is it unused errors (nowhere in the code), deprecated errors (still present in the code) or both? (I think it's both.)